### PR TITLE
Setup release from tags

### PR DESCRIPTION
**Pull request by timja-kainos**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
 script:
-  - yarn --version
-  - yarn check --integrity
-  - yarn test
+- yarn --version
+- yarn check --integrity
+- yarn test
 before_install:
-  - yarn global add greenkeeper-lockfile@1
+- yarn global add greenkeeper-lockfile@1
```
Comment (doncem) [inline .travis.yml]: don't remove correct spacing in yaml file

Comment (timja-kainos) [reply .travis.yml]: hmm, this wasn't touched by hand only changed by the `travis` command line tool.
I would side with travis as this is a file for their CI

Comment (doncem) [reply .travis.yml]: oh. normally i would trust travis cli but will not accept their own "spacing". it is under developed

Comment (timja-kainos) [reply .travis.yml]: problem is if you use their tool to change anything in the file it will just change it back, why fight?

Comment (timja-kainos) [reply .travis.yml]: I just tested by manually adding spaces back and then running:
```
travis encrypt  --add
```
and the spaces were removed

Comment (doncem) [reply .travis.yml]: no fight. your explanation was enough

Comment (timja-kainos) [reply .travis.yml]: I didn't mean me fight you, I meant why fight the tool.
Sorry if any misunderstanding 👍

Comment (doncem) [reply .travis.yml]: i understood it correctly ;)

**Pull request merged**